.github/workflows/build-and-test-stack.yml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch: { }
   schedule:
     # Every Sunday at 1:00 AM
```

Comment:
Can you please edit the commented line to be in accordance with what the cron does?